src/components/Participant/RegistrationProcess/StartRegistrationProcess/StartRegistrationProcess.tsx

```diff
@@ -2,13 +2,40 @@ import DocumentTitle from "react-document-title";
 import StepWrapper from "../../../Shared/StepWrapper/StepWrapper";
 import { ContentContext } from "../../../../context/ContentContext";
 import { useContext } from "react";
+import { Stack } from "@mui/material";
+import DTEButton from "../../../Shared/UI/DTEButton/DTEButton";
+import { useHistory } from "react-router-dom";
+import NhsLoginButton from "../../../Shared/UI/NhsLoginButton";
 
 function StartRegistrationProcess() {
   const { content } = useContext(ContentContext);
+  const history = useHistory();
+
+  const handleGoBack = () => {
+    history.goBack();
+  };
 
   return (
     <DocumentTitle title={content["register-page-document-title"]}>
-      <StepWrapper>{content["register-page"]}</StepWrapper>
+      <StepWrapper>
+        {content["register-page"]}
+        <Stack spacing={2} direction="row">
+          <NhsLoginButton buttonText="Continue to NHS login" />
+          <a href="https://volunteer.bepartofresearch.nihr.ac.uk/Participants/Register/Questions">
```

Comment:
React router uses the base url out of the box.  Hardcoding like this will take you out of localhost:300 and onto a different environment